client/src/LandingPage.js

```diff
@@ -66,9 +76,18 @@ function LandingPage() {
                   value={character}
                   onChange={(e) => setCharacter(e.target.value)}
                 >
-                  <option className="option-placeholder" value="" disabled>
+                  <option hidden value="">
                     Select your character
                   </option>
+                  {account.characters.map(
+                    ({ name, level, class: clazz, league }) => {
```

Comment:
i just wondering why we are using clazz as the temp variable for class,  educate me if the naming is okay.

Reply:
class is a reserved keyword, so it can't be used as a variable name. Common conventions use clazz or cls instead.

Reply:
sounds good, thanks for letting me know.